modules/dapps.js

```diff
@@ -281,8 +282,13 @@ DApps.prototype.shared = {
  * @todo Add description of the function
  */
 shared.getGenesis = function(req, cb, tx) {
-	(tx || library.db).dapps
-		.getGenesis(req.dappid)
+	library.storage.entities.Transaction.get(
+		{
+			id: req.dappid,
+		},
+		{},
+		tx
+	)
```

Comment:
Maybe it's a good idea to use `limit: 1` and `sort: rowId:asc`

Also, in the line 301 it's using `authorId`. Replace it by `senderId`, pls.

Reply:
Done and Done. Resolving